tools/loader/ImageClassifier.cpp

```diff
@@ -83,11 +83,18 @@ llvm::cl::opt<ImageNormalizationMode> imageMode(
 llvm::cl::alias imageModeA("i", llvm::cl::desc("Alias for -image_mode"),
                            llvm::cl::aliasopt(imageMode),
                            llvm::cl::cat(imageLoaderCat));
+
+
+llvm::cl::opt<std::string> loadimageFormat(
```

Comment:
We tend to use camelCase for variable names. Also, "format" is very general word. Let's use "layout"

loadimageFormat -> imageLayout

Reply:
Will do